uwmac0108.xml

```diff
@@ -512,11 +638,54 @@
         </c02>
         <c02>
           <did>
+            <unittitle>Self-Study, 1998-1999</unittitle>
+            <container type="box" label="Box">8</container>
+            <container type="folder" label="Folder">27</container>
+          </did>
+        </c02>
+        <c02>
+          <did>
             <unittitle>Syllabi</unittitle>
           </did>
           <c03>
             <did>
               <unittitle>
+                <unitdate>1988-2010</unitdate>
+              </unittitle>
+              <container type="box" label="Box">9</container>
+              <container type="folders" label="Folders">1-29</container>
+            </did>
+          </c03>
+          <c03>
+            <did>
+              <unittitle>
+                <unitdate>1988-2010</unitdate>
+              </unittitle>
+              <container type="box" label="Box">11</container>
+              <container type="folders" label="Folders">1-10</container>
+            </did>
+          </c03>
+          <c03>
+            <did>
+              <unittitle>
+                <unitdate>1989-2010</unitdate>
```

Comment:
Are these arranged in the boxes by course number? If so, we should indicate such in the contents list. This doesn't make a lot of sense to me as is (why is box 11 before box 10?).